msteams-platform/graph-api/activity-feed/test-activity-feed-notifications.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Testing activity feed notifications in Teams
+description: Details testing activity feed notifications in Teams using Postman
+localization_priority:  Normal
+author: laujan
+ms.author: lajanuar
+ms.topic: How-to
+keywords: teams authorization OAuth SSO AAD activity feed notification Postman Graph
+---
+
+# Test activity feed notification in Teams
+
+The activity feed API is a Microsoft Teams and Microsoft Graph integration that enables your app to use API endpoints to send activity feed notifications to Teams users. Activity feed notifications can range from identifying tasks requiring a user's attention, to actions taken by others that affect the user or a set of users.
+
+> [!NOTE]
+>To test activity feed notifications, your Teams app manifest file must include **webApplicationInfo** and **activities** keys. *See*, [Update your Teams app manifest](feed-notifications.md#update-your-teams-app-manifest).
+
+## Test activity feed notifications using the Postman app
+
+To check whether the activity feed notifications are being honored by the API request, you'll need to copy the [Activity feed notification test code](test-activity-feed-json-file.md) into your local environment and update the following values:
+
+1. `client_id`  — your app's Azure AD app id.
+1. `client_secret`  — your Azure AD app secret (password)
+1. `tenantId` — you can get the unique identifier tenant id from the Teams client as follows:
+
+> [!div class="checklist"]
+>
+> * In the Teams client, select **Teams** from the far left nav bar .
+> * Select the team where the app is installed from the dropdown menu.
+> * Select the **More options** icon (&#8943;).
+> * Select **Get link to team**.
+> * Copy and save the **tenantId** value from the string:
+
+&emsp;&emsp; ![find tenant ID in Teams](../../assets/images/find-tenant-id.png)
```

Comment:
In the Postman collection, the tenantId would actually be more pertinent for the authentication calls. For example, in 
'https://login.microsoftonline.com/**60bcdb18-3e8b-4c8d-8e03-c9bd0a442378**/oauth2/v2.0/token', the **bolded** part is where the caller's tenant would be. 

For the above image, I would say do two things:
1) In the JSON (here: https://review.docs.microsoft.com/en-us/microsoftteams/platform/graph-api/activity-feed/test-activity-feed-json-file?branch=activity-feed-documentation), I would remove the '&tenantId=...' parameter altogether because it's not required. That way people won't get confused between the authentication tenantId and this other one.
2) Remove the image to limit confusion.